docs/data/routes/docs/nextjs/getting-started/walkthrough-jsscreate/en.md

```diff
@@ -3,4 +3,67 @@ name: walkthrough-jsscreate
 routeTemplate: ./data/component-templates/article.yml
 title: Walkthrough with `jss create`
 ---
-# Walkthrough: with `jss create`
\ No newline at end of file
+# Walkthrough: Using the JSS CLI to get started with JSS and Next.js 
+
+You can create a JSS Next.js application by using the JSS command-line tool (CLI). 
+
+> When using this approach, you will need to perform additional steps to connect to a Sitecore instance.
+
+We recommend this approach for Sitecore developers that use the [code-first developer workflow](https://jss.sitecore.com/docs/fundamentals/dev-workflows/code-first). 
+
+## TL;DR
+
+To quickly scaffold a JSS Next.js application, run the following commands in a terminal:
+
+```
+npm install -g @sitecore-jss/sitecore-jss-cli
+jss create my-first-jss-app nextjs
+cd my-first-jss-app
+jss start
+```
+
+##  Create your application
+
+To create a JSS application, you must: 
+
+1. Install  [Node.js](https://nodejs.org/).
+
+2. Install the JSS CLI: 
+
+   ```
+   npm install -g @sitecore-jss/sitecore-jss-cli
+   ```
+
+   > To verify the successful installation of the JSS CLI and to see the list of commands availably globally, run `jss --help`.
+
+3. Run:
+
+   ```
+    jss create my-first-jss-app nextjs
+   ```
+
+   The command will create the JSS Next.js application and install the required packages. 
+
+## Run your application
+
+To run your application: 
+
+1. In the terminal, go to your application directory. For example: `cd my-first-jss-app`.
+
+2. Run `jss start`.
+
+   > To see the full list of commands available for your application, run `jss --help`.
+
+## Connect to Sitecore
+
+Pulling content in your JSS application from Sitecore  and using JSS in production, will require a Sitecore instance. 
+
+Sitecore requires Windows, but the instance can be on a virtual machine or a remote server. 
+
+To connect your application to Sitecore, you must:
+
+1. [Setup JSS Server Components](https://jss.sitecore.com/docs/getting-started/jss-server-install).
```

Comment:
Yeah I think linking to these for now is good, once we restructure all the docs to focus on Next.js, we can move / reword these articles as well.